float64 Switch
float64[5] Wave
float64 Temp
float64 Humidity
float64 PM_25